src/main/java/fact/utils/ElementwiseMultiplyDoubleArray.java

```diff
@@ -0,0 +1,46 @@
+package fact.utils;
+
+import stream.Data;
+import stream.Processor;
+import stream.annotations.Parameter;
+import fact.features.singlePulse.timeSeriesExtraction.ElementWise;
```

Comment:
I allready approved ;)

I had the discussion with @relleums that his ElementWise functions might be so usefull, that they should go in a more general place. And here is the first class, which uses them outside the singlePulse package.

So we should think about moving them.